MdePkg/Test/Mock/Library/GoogleTest/Ppi/MockReadOnlyVariable2.cpp

```diff
@@ -0,0 +1,21 @@
+/** @file MockReadOnlyVariable2.cpp
+  Google Test mocks for PeiServicesLib

```

Comment:
This says it is a mock of PeiServicesLib, although this contains just the read only variable PPI.

A general question would be whether the model is to mock the library or just the interface, separately?